README.md

```diff
@@ -44,159 +40,158 @@ In addition, the file name should be of the format `time-report-x.csv`,
-1. A `Pay Period` is a date interval that is roughly biweekly. Each month has
-   two pay periods; the _first half_ is from the 1st to the 15th inclusive, and
-   the _second half_ is from the 16th to the end of the month, inclusive.
-1. Each employee should have a single row in the report for each pay period
-   that they have recorded hours worked. The `Amount Paid` should be reported
-   as the sum of the hours worked in that pay period multiplied by the hourly
-   rate for their job group.
-1. If an employee was not paid in a specific pay period, there should not be a
-   row for that employee + pay period combination in the report.
-1. The report should be sorted in some sensical order (e.g. sorted by employee
-   id and then pay period start.)
-1. The report should be based on all _of the data_ across _all of the uploaded
-   time reports_, for all time.
-
-As an example, a sample file with the following data:
-
-<table>
-<tr>
-  <th>
-    date
-  </th>
-  <th>
-    hours worked
-  </th>
-  <th>
-    employee id
-  </th>
-  <th>
-    job group
-  </th>
-</tr>
-<tr>
-  <td>
-    4/11/2016
-  </td>
-  <td>
-    10
-  </td>
-  <td>
-    1
-  </td>
-  <td>
-    A
-  </td>
-</tr>
-<tr>
-  <td>
-    14/11/2016
-  </td>
-  <td>
-    5
-  </td>
-  <td>
-    1
-  </td>
-  <td>
-    A
-  </td>
-</tr>
-<tr>
-  <td>
-    20/11/2016
-  </td>
-  <td>
-    3
-  </td>
-  <td>
-    2
-  </td>
-  <td>
-    B
-  </td>
-</tr>
-</table>
-
-should produce the following payroll report:
-
-<table>
-<tr>
-  <th>
-    Employee ID
-  </th>
-  <th>
-    Pay Period
-  </th>
-  <th>
-    Amount Paid
-  </th>
-</tr>
-<tr>
-  <td>
-    1
-  </td>
-  <td>
-    1/11/2016 - 15/11/2016
-  </td>
-  <td>
-    $300.00
-  </td>
-</tr>
-  <td>
-    2
-  </td>
-  <td>
-    16/11/2016 - 30/11/2016
-  </td>
-  <td>
-    $90.00
-  </td>
-</tr>
-</table>
-
-Your application should be easy to set up, and should run on either Linux or
-Mac OS X. It should not require any non open-source software.
-
-There are many ways that this application could be built; we ask that you build
-it in a way that showcases one of your strengths. If you enjoy front-end
-development, do something interesting with the interface. If you like
-object-oriented design, feel free to dive deeper into the domain model of this
-problem. We're happy to tweak the requirements slightly if it helps you show
-off one of your strengths.
+We've agreed to build an API with the following endpoints to serve HTTP requests:
+
+1. An endpoint for uploading a file.
+
+   - This file will conform to the CSV specifications outlined in the previous section.
+   - Upon upload, the timekeeping information within the file must be stored to a database for archival purposes.
+   - If an attempt is made to upload a file with the same report ID as a previously uploaded file, this upload should fail with an error message indicating that this is not allowed.
+
+1. An endpoint for retrieving a payroll report structured in the following way:
```

Comment:
Should we consider an optional start/end date or pay period param for filtering this endpoint? It slightly increases the complexity but I think gives us something to look at during the code evaluation.

Reply:
Leaving this out for now. I'll bring it up with the working group in our next meeting.